android-components/components/support/utils/src/main/res/drawable/ic_cc_logo_amex.xml

```diff
@@ -1,3 +1,7 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

Comment:
It was intentional that we did not include our license for these `ic_cc_<>.xml` vectors. The reasoning is that these are credit card logos which we don't own. Is there a way to allow these files to avoid the linter?

Reply:
No problem; when the time comes, we'll add these files to the linter exclude list.